valhalla/mjolnir/osmway.h

```diff
@@ -1727,6 +1741,9 @@ struct OSMWay {
   // Truck speed in kilometers per hour
   uint8_t truck_speed_;
 
+  // Z-level of the way relatively to other levels
+  int8_t z_level_;
```

Comment:
you should rename the spare below and use that instead

Reply:
okay, will do. Could you elaborate here? I cannot get why do we have this spare field. Do we serialize ways somewhere?